Jenkinsfile.deploy

```diff
@@ -200,7 +203,7 @@ pipeline {
             }
         }
 
-        stage('Create PRs'){
+        stage('Create PRs') {
```

Comment:
so we prepare for Pr and update quickstarts, but we don't create any PR for quickstarts ?

Reply:
I was not sure if we need to reset stable branch here or not so planned to split task so to change versions only, apparently I need to change it in promote so I decided to finish with pr here. I added changes to https://github.com/dupliaka/optaplanner/tree/TTPLANNER-2230
But testing is blocked by https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/129
So didnt merge it yet, sorry do not how to convert PR to template.

Reply:
Resetting the stable branch must happen only after we are sure the release is successful, thus it must happen in the "promote" phase. Here we just have to prepare a PR to the release branch, e.g. `8.1.x`.

Reply:
Usually, we change the version in `deploy` script and then create a PR that we save in `deployment.properties` file.
Then in `promote` script, we merge the PR.
Anyway here, the changes will be applied to release branch and not `stable` branch.
If then you need to update `stable` branch, please check what we do in [kogito-examples](https://github.com/kiegroup/kogito-examples/blob/stable/Jenkinsfile.promote#L91-L93)

Reply:
As I can see, kogito-examples already `forcePushProtectedBranch`, which is something we need to do as well.